src/components/ScreenWrapper/index.js

```diff
@@ -109,7 +111,7 @@ class ScreenWrapper extends React.Component {
                                 paddingStyle,
                             ]}
                         >
-                            <KeyboardAvoidingView style={[styles.w100, styles.h100, {maxHeight: this.props.windowHeight}]} behavior={this.props.keyboardAvoidingViewBehavior}>
+                            <KeyboardAvoidingView style={[styles.w100, styles.h100, {maxHeight}]} behavior={this.props.keyboardAvoidingViewBehavior}>
```

Comment:
Why don't we depend on `Dimensions` (`windowHeight`) inside `KeyboardAvoidingView/index.web.js` and apply `maxHeight` there? That would be a clean symmetry to the `index.ios.js` implementation and would reflect the general contract of the built-in `KeyboardAvoidingView`.

The `shouldEnableMaxHeight` would need to be forwarded to the in-house `KeyboardAvoidingView`, but would be accessed only on the Web implementation, which would improve clarity. Because the `maxHeight` thing _is_ only for the Web, right?

I know that this PR does not introduce this code, but it does the biggest changes around it in a while, so if the original implementation was created in a way I'd describe as "confusing", I think we should clean it up here.

Reply:
Then, we'd also have a good place to comment on which browsers actually need that.